code/implementation/docker/machine/src/main/java/io/cattle/platform/docker/machine/launch/WebsocketProxyLauncher.java

```diff
@@ -51,12 +51,14 @@
 
     protected void prepareConfigFile() throws IOException {
         String config = API_FILTER_PROXY_CONFIG.get();
+        String configContent;
         if (StringUtils.isBlank(config)) {
-            new File(API_FILTER_PROXY_CONFIG_FILE.get()).delete();
+            configContent = "{}";
```

Comment:
Your service shouldn't assume a file exists.  I'd rather we delete the file if the config is blank.